### ISSUE TITLE
preserve or retrieve code bundle 

### ISSUE THREAD

**Issue by benedicthsieh** [Microsoft/react-native-code-push#646]
We'd like to get the code bundle that is generated by release-react so that we can upload it to Sentry. We see that the bundle is put into a temporary directory, but it seems like that directory is cleaned up once the release is done. Is there a way to either retrieve the bundle afterwards (maybe with a code-push CL command), or preserve the output bundle that's created by release-react?

Thanks!

**Comment by vladimir-kotikov** [comment]
@benedicthsieh, currently there is no such option - temp directory gets wiped after successful release. However `release-react` runs `react-native bundle --assets-dest /output-dir --bundle-output /output-dir/main.jsbundle --dev false --entry-file index.ios.js --platform ios` to generate updated bundle, so you could try to reuse this command to get the same bundle as in release

**Comment by benedicthsieh** [comment]
Thanks for the speedy response, that's the solution we'll adopt in the near term.

Would the team consider adding a `bundleOutput` flag to `release-react` (like the existing `sourcemapOutput` flag) for the next release? Or maybe `code-push app get <version>` in case people wanted to archive their bundles elsewhere?

**Comment by vladimir-kotikov** [comment]
Thanks for suggestion - this might be a useful feature 👍 
/cc @silhouettes to help decide whether this should be added to backlog

**Comment by panarasi** [comment]
@vladimir-kotikov Can you create a new issue capturing the need to in the code-push repo and then close this issue please ?

**Comment by vladimir-kotikov** [comment]
@panarasi, done!
